src/audio.c

```diff
@@ -258,6 +256,13 @@ void audio_init(const int frequency, int numbuffers, audio_fill_buffer_callback
     _paused = false;
 }
 
+void audio_set_buffer_callback(audio_fill_buffer_callback fill_buffer_callback)
+{
+    disable_interrupts();
+    _fill_buffer_callback = _orig_fill_buffer_callback = fill_buffer_callback;
```

Comment:
Hate to nitpick, but setting this when paused would mean we would unpause, right? Probably low-pri but still nice to be correct.